tensorflow/c/tf_tensor.h

```diff
@@ -55,6 +55,7 @@ extern "C" {
 // Allocator Attributes used for tensor allocation. 
 typedef struct TF_AllocatorAttributes { 
   size_t struct_size; 
+  // Set flag to 0 for CPU allocation, else 1. 
```

Comment:
To be precise, this is a boolean, not a flag.

Reply:
Thanks for pointing that out!